sys/dev/random/fortuna.c

```diff
@@ -231,19 +232,36 @@ random_fortuna_process_event(struct harvest_event *event)
 	 * We must be locked against pool state modification which can happen
 	 * during accumulation/reseeding and reading/regating.
 	 */
+
+
 	pl = event->he_destination % RANDOM_FORTUNA_NPOOLS;
-	randomdev_hash_iterate(&fortuna_state.fs_pool[pl].fsp_hash, event, sizeof(*event));
+	
+	/* we toss low entropy static/counter fields towards the end of the 
+	 * he_event structure in order to increase measurable entropy when
+	 * conducting SP800-90B entropy analysis measurements of seed material fed into PRNG
+	 *
+	 *  -- wdf
+	 */
+	uint32_t entropy_data_size = 0;
+	if (event->he_size == 4)
+		entropy_data_size = 8;
+	else if (event->he_size == 8)
```

Comment:
What's in else case, when the event->he_size if not 4 nor 8?
At least it would be nice to add an else panic() or at lease a KASSERT.